### DISCUSSION TITLE
How to log yaml config when using the  WandbLogger?

### DISCUSSION THREAD

**Discussion by celsofranssa** [Lightning-AI/pytorch-lightning#20259] [General]
I am using the [WandbLogger](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.loggers.wandb.html#lightning.pytorch.loggers.wandb.WandbLogger), and I'd like to log (or save as artifact) the [hydra config](https://hydra.cc/docs/intro/) (such as the one shown below) used to configure all the fit, predict, and test steps in the same wandb run where is logged the he metrics, loss, etc. 

What is the best approach to accomplish this?

```yaml
db:
  driver: mysql
  user: omry
  pass: secret
ui:
  windows:
    create_db: true
    view: true
schema:
  database: school
  tables:
  - name: students
    fields:
    - name: string
    - class: int
  - name: exams
    fields:
    - profession: string
    - time: data
    - class: int
````
